torchvision/ops/focal_loss.py

```diff
@@ -10,25 +10,25 @@ def sigmoid_focal_loss(
     alpha: float = 0.25,
     gamma: float = 2,
     reduction: str = "none",
-):
+) -> torch.Tensor:
     """
     Original implementation from https://github.com/facebookresearch/fvcore/blob/master/fvcore/nn/focal_loss.py .
```

Comment:
Ad discussed at https://github.com/pytorch/vision/pull/5776#discussion_r844892807, long string URLs cause rendering issues on the documentation. Let's move this to a comment on source code.
```suggestion
```